### ISSUE TITLE
Max Mint per Address (?)

### ISSUE THREAD

**Issue by adamkrellenstein** [CounterpartyXCP/counterparty-core#2698]

**Comment by droplister** [comment]
I think the main thing this achieves is the same number of individuals minting from more addresses than they otherwise would. Address re-use within Counterparty, Ethereum, Solana, Runes, is so popular and common that its not worth adding this setting.

**Comment by adamkrellenstein** [comment]
So the only advantage I can think of here is that it adds a layer of pseudonymity: it makes it *look* fair. OTOH that matters.

**Comment by adamkrellenstein** [comment]
Yep: https://github.com/CounterpartyXCP/counterparty-core/issues/3006
